types/googlepay/index.d.ts

```diff
@@ -1146,6 +1172,45 @@ declare namespace google.payments.api {
         token: string;
     }
 
+    /**
+     * Definition of merchant provided offers that may be applicable to the
+     * current order.
+     */
+    interface OfferInfo {
+        /**
+         * List of merchant provided offers applicable to the current order.
+         */
+        offers: OfferDetail[];
+    }
+
+    /**
+     * Definition for each offer to be applied to this Payment Request.
+     */
+    interface OfferDetail {
+        /**
+         * Redemption code available for this transaction. This would be used to
+         * identify the offer in case the user decides to apply the offer.
+         */
+        redemptionCode: string;
+
+        /**
+         * Description for the offer visible to the user to inform them about
+         * the offer. The description would be displayed in buyflow and should
+         * be less than 60 characters long.
+         */
+        description: string;
+    }
+
+    /**
+     * Definition for each offer to be applied to this Payment Request.
```

Comment:
If it's not a particular "Payment Request", I'd suggest we "lowercase it".

Reply:
Will apply in a separate PR